package_mrtrix

```diff
@@ -62,7 +61,7 @@ EOF
     )
 
     echo - collating dependent system libraries...
-    LIBS=$(for n in release/bin/*; do ldd $n; done | sed 's/(.*)$//g' | sed 's/^.*=>//g' | sort | uniq | grep -v 'mrtrix\|vdso\|libGL.so\|libEGL.so\|nvidia\|fglrx' | xargs)
+    LIBS=$(for n in bin/*; do ldd $n; done | sed 's/(.*)$//g' | sed 's/^.*=>//g' | sort | uniq | grep -v 'mrtrix\|vdso\|libGL.so\|libEGL.so\|nvidia\|fglrx' | xargs)
```

Comment:
For scripts, ldd produces the output "not a dynamic executable" which triggers the warning "not all executables have dynamic libraries. Possibly skipping dependent libraries for some commands. This is expected if you built with -static" 

I'd just get rid of the warning. Alternatively, we could use `file` to filter out scripts.